go/vt/vttablet/endtoend/framework/client.go

```diff
@@ -238,21 +238,13 @@ func (client *QueryClient) StreamExecute(query string, bindvars map[string]*quer
 // StreamExecuteWithOptions executes a query & returns the results using 'options'.
 func (client *QueryClient) StreamExecuteWithOptions(query string, bindvars map[string]*querypb.BindVariable, options *querypb.ExecuteOptions) (*sqltypes.Result, error) {
 	result := &sqltypes.Result{}
-	err := client.server.StreamExecute(
-		client.ctx,
-		client.target,
-		query,
-		bindvars,
-		0,
-		options,
-		func(res *sqltypes.Result) error {
-			if result.Fields == nil {
-				result.Fields = res.Fields
-			}
-			result.Rows = append(result.Rows, res.Rows...)
-			return nil
-		},
-	)
+	err := client.server.StreamExecute(client.ctx, client.target, query, bindvars, 0, 0, options, func(res *sqltypes.Result) error {
```

Comment:
nit: these lines become too long after putting them all into a single line. 133 characters is a lot